design/nab_status_update.md

```diff
@@ -0,0 +1,111 @@
+
+Those are are the possible values for phase:
+
+| **Value** | **Description**                 |
+|-----------|--------------------------------|
+| New | *NonAdminBackup resource was accepted by the OpenShift cluster, but it has not yet been processed by the NonAdminController* |
+| BackingOff | Velero *Backup* object was not created due to NonAdminBackup error (configuration or similar) |
+| Created | Velero *Backup* was created. The Phase will not have additional informations about the |
+
+### Conditions
+
+The `conditions` is also a part of the NonAdminBackup's `status` field. One NAB object may have multiple conditions. It is more granular knowledge of the NonAdminBackup object and represents the array of the conditions through which the NonAdminBackup has or has not passed. Each `NonAdminCondition` has one of the following `type`:
+
+| **Condition** | **Description**                 |
+|-----------|--------------------------------|
+| Accepted | The Backup object was accepted by the reconcile loop, but the Velero Backup may have not yet been created |
+| Queued | The Velero Backup was created succesfully. At this stage errors may still occur either from the Velero not accepting backup or during backup procedure. |
+
+The `condition` data is also accomapied with the following:
+
+| **Field name** | **Description**                 |
+|-----------|--------------------------------|
+| type | The `Type` of the condition |
+| status | represents the state of individual condition. The resulting `phase` value should report `Success` only when all of the conditions are met and the backup succeded. One of `True`, `False` or `Unknown`. |
+| lastTransitionTime | Timestamp for when the NonAdminBackup last transitioned from one status to another. |
+| reason | Machine-readable, UpperCamelCase text indicating the reason for the condition's last transition. |
+| message | Human-readable message indicating details about the last status transition. |
+
+### VeleroBackupStatus
+
+The `VeleroBackupStatus` that is part of `NonAdminBackup.Status` is a copy of the `VeleroBackup.Status`.
+
+
+> Sample `VeleroBackupStatus` with `Conditions` of a `NonAdminBackup` object that allowed to create `VeleroBackup` object, but there was an error while performing backup itself:
+
+```yaml
+status:
+  veleroBackupStatus:
+    expiration: '2024-05-16T08:12:11Z'
+    failureReason: >-
+      unable to get credentials: unable to get key for secret: Secret
+      "cloud-credentials" not found
+    formatVersion: 1.1.0
+    phase: Failed
+    startTimestamp: '2024-04-16T08:12:11Z'
+    version: 1
+  conditions:
+    - lastTransitionTime: '2024-04-15T20:27:35Z'
+      message: Backup was accepted by the NAC controller
+      reason: backup_accepted
+      status: 'True'
+      type: Accepted
+    - lastTransitionTime: '2024-04-15T20:27:45Z'
+      message: Created Velero Backup object
+      reason: backup_scheduled
```

Comment:
both reasons do not follow description: UpperCamelCase

Reply:
Changed.